packages/radix-ui-themes/src/components/flex.props.ts

```diff
@@ -1,5 +1,5 @@
-import { asChildProp, gapProps } from '../helpers';
-import type { PropDef } from '../helpers';
+import { asChildProp, gapProps } from '../helpers/index.js';
+import type { PropDef } from '../helpers/index.js';
```

Comment:
This is so odd to read haha, a type import from a JS file 😀

Reply:
I know, breaks my brain too